packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -162,7 +184,12 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
         icon:
           'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Fsimplex.jpg?alt=media&token=6037b2f9-9d76-4076-b29e-b7e0de0b3f34',
         image: <Image source={simplexLogo} style={styles.logo} resizeMode={'contain'} />,
-        onSelected: () => openSimplex(account),
+        isFeeDataLoading: !simplexQuote?.quote_id,
```

Comment:
Should we use `asyncSimplexQuote.loading` instead? Otherwise, if the Simplex API call fails we'll have an infinite loading here.